tests/jerry/date-parse.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2015 Samsung Electronics Co., Ltd.
+// Copyright 2015 University of Szeged.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var wrongFormats = ["", "2", "20", "201", "2015-", "2015-01-", "2015-01-01-",
+                    "qwerty", "2015-01-01T", "2015-01-01T1:1", "2015-01-01T01",
+                    "2015-01-01T01", "2015-01-01T01:01F", "T2015", "2015-01-01Z",
+                    "2015-01-01+01:00", "2015-01-01T00:00+01", "2015-01-01T00:00+1",
+                    "2015-01-01T00:00-01", "2015-01-01T00:00.000", "2015-01-01T00:00:",
+                    "2015-01-01T00:", "2015-01-01T00:00:00.1", "2015-01-01T00:00:00.01",
+                    "2015-01-01T00:00+01:00Z", "2015/01/01", "2015-01-32", "2015--1",
+                    "2015-13", "2015-01--1", "-215", "-215-01-01", "2015-01-00",
+                    "2015-01-01T25:00", "2015-01-01T00:60", "2015-01-01T-1:00",
+                    "2015-01-01T00:-1"];
```

Comment:
Could we have this as one entry per line? Also maybe some other types like undefined, numbers, objects?